### DISCUSSION TITLE
Decompiled characters have broken skeletons. Is there a fix?

### DISCUSSION THREAD

**Discussion by Santiago-bupo** [SteamDatabase/ValveResourceFormat#814] [General]
I wanted to decompile the gendarmerie models so I could photoshop their textures and make custom playermodels for my maps.
But when I decompiled the models, there were bones, morphs, IK and other values and parameters missing. This made the model unusable of course.
Is there a way to fix this? I did try to copy and paste some missing values from uncompiled agents but to no avail.
Since I didn't intend to change the mesh model, I tried only decompiling the materials, modifying them and putting them in my addon but it seems it doesn't overwrite the original textures.
![imagen_2024-06-25_122205212](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/107816611/ac1f2a78-626b-4bb7-818d-f3052919e708)

**Comment by Lacrimossa8** [comment]
Still no fix for it? Same problem after decompile
